telemetry/definitions/commonDefinitions.json

```diff
@@ -855,6 +855,71 @@
             "name": "userChoice",
             "type": "string",
             "description": "User selection from a predefined menu (not user-provided input)" 
+        },
+        {
+            "name": "awsqAssignClickCount",
```

Comment:
@justinmk3 Is there a defined convention on these names? We're following how codewhisperer formats their names (camal case), but other teams like codecatalyst seem to prefix with `codecatalyst_${name}`

Reply:
The codecatalyst names were an accident, I think. Probably best to continue using `topicCamelCase` for types. Whereas metric names use `topic_camelCase`.

Reply:
@justinmk3  can I stil awsq_ or is awsQ_ is expected?

Reply:
`awsq_`. sorry for confusion, updated above comment.